llvm/unittests/Analysis/TargetLibraryInfoTest.cpp

```diff
@@ -621,3 +622,62 @@ TEST_F(TargetLibraryInfoTest, ValidProto) {
     EXPECT_TRUE(isLibFunc(F, LF));
   }
 }
+
+namespace {
+
+// Creates TLI for AArch64 and VecLibrary ARmPL, and uses it to get the TLI
```

Comment:
Typo "ARmPL"

That said, what's the relevance to a vector library given the test only relates to scalar functions?